lib/tasks/forem.rake

```diff
@@ -3,8 +3,7 @@ namespace :forem do
   # Since we execute this tasks in bin/setup as well as app_initializer:setup
   # everything happening here needs to be idempotent.
   task setup: :environment do
-    # NOTE: There are currently no tasks here. This will change as soon as we
-    # add/update the task for seeding navigation links to support RFC #237.
+    Rake::Task["navigation_links:create"].invoke
```

Comment:
double checked this was idempotent by destroying all existing navigation links, then calling the rake task 3 times, and listing the links.

```
[2] pry(main)> NavigationLink.destroy_all
# lots of DELETE FROM "navigation_links" statements here

[3] pry(main)> NavigationLink.all.pluck(:position, :section, :name)
   (0.6ms)  SELECT "navigation_links"."position", "navigation_links"."section", "navigation_links"."name" FROM "navigation_links"
=> []
[4] pry(main)> 

djuber@forem:~/src/forem$ bundle exec rake navigation_links:create
djuber@forem:~/src/forem$ bundle exec rake navigation_links:create
djuber@forem:~/src/forem$ bundle exec rake navigation_links:create
djuber@forem:~/src/forem$ bundle exec rails consoleRunning via Spring preloader in process 308549
Loading development environment (Rails 6.1.4)
[1] pry(main)> NavigationLink.all.pluck(:position, :section, :name)
   (0.7ms)  SELECT "navigation_links"."position", "navigation_links"."section", "navigation_links"."name" FROM "navigation_links"
=> [[0, "default", "Reading List"],
 [1, "default", "Contact"],
 [0, "other", "Code of Conduct"],
 [1, "other", "Privacy Policy"],
 [2, "other", "Terms of use"]]
```